aioftp/server.py

```diff
@@ -857,6 +858,9 @@ def __init__(self,
             loop=self.loop
         )
         response_queue = asyncio.Queue(loop=self.loop)
+        IS_PY37 = sys.version_info[:2] == (3, 7)
+        current_task = asyncio.current_task if IS_PY37 else asyncio.Task.current_task
```

Comment:
`current_task` should be named as `get_current_task`, I think.

Reply:
Done!